install/kubernetes/helm/istio/values.yaml

```diff
@@ -80,6 +80,10 @@ global:
   # propagated, not recommended for tests.
   controlPlaneSecurityEnabled: false
 
+  # SDS API enabled. Will result in distribute key cert through SDS(secret discovery service)
+  # instread of secret-mount files.
+  sdsAPIEnabled: false
```

Comment:
sdsEnabled ( API doesn't make sense - a lot of things are APIs ).

But I would rather use

sds:
   enabled: false

I would expect other sds-specific settings will show up ( refresh interval, etc ).